_source/_posts/2023-07-31-workflows-workshop.md

```diff
@@ -0,0 +1,939 @@
+---
+layout: blog_post
+title: "Enterprise Ready Workshop: Automate with with no-code Okta Workflows"
+author: max-katz
+by: advocate
+communities: [no-code,javascript]
+description: "Add automated reporting with no-code Okta Workflows. Enhance customer satisfaction by enabling custom automation."
+tags: [enterprise-ready-workshops, workflows, no-code, low-code]
```

Comment:
@maxkatz please work with @edunham on these tags. they are changing.

Reply:
@edunham please suggest what tags I need to use in the Workflows workshop.

Reply:
I set the following tags: 
`[enterprise-ready-workshops, react, express, oidc, workflows, automation, no-code, low-code]`